tools/Azure.Mcp.Tools.AppService/tests/test-resources.bicep

```diff
@@ -136,26 +136,6 @@ resource sqlServer 'Microsoft.Sql/servers@2023-05-01-preview' = {
   }
 }
 
-// SQL DB Contributor role definition
-resource sqlDbContributorRoleDefinition 'Microsoft.Authorization/roleDefinitions@2018-01-01-preview' existing = {
-  scope: subscription()
-  // This is the SQL DB Contributor role
-  // Lets you manage SQL databases, but not access to them
-  // See https://learn.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#sql-db-contributor
-  name: '9b7fa17d-e63e-47b0-bb0a-15c516ac86ec'
-}
-
-// Role assignment for test application
-resource appSqlRoleAssignment 'Microsoft.Authorization/roleAssignments@2022-04-01' = {
```

Comment:
Do we not need role assignment for test application anymore?

Reply:
The same role assignment exists at the bottom of the file.
scope: sqlServer
principalId: testApplicationOid

the only difference is the second assignment uses an inline computed role definition id instead of pulling the id from the role definition resource:

```
roleDefinitionId: sqlDbContributorRoleDefinition.id
```
where 
```
sqlDbContributorRoleDefinition = {
  scope: subscription()
...
  name: '9b7fa17d-e63e-47b0-bb0a-15c516ac86ec'
}
```
vs
```
roleDefinitionId: subscriptionResourceId('Microsoft.Authorization/roleDefinitions', '9b7fa17d-e63e-47b0-bb0a-15c516ac86ec') // SQL DB Contributor
```